manifest_staging/charts/csi-secrets-store-provider-azure/templates/role.yaml

```diff
@@ -0,0 +1,13 @@
+{{- if .Values.rbac.pspEnabled }}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: {{ template "sscdpa.fullname" . }}
+  namespace: {{ .Release.Namespace }}
+rules:
+  - apiGroups: [ 'extensions' ]
+    resources: [ 'podsecuritypolicies' ]
+    verbs: [ 'use' ]
+    resourceNames:
+      - secrets-store
```

Comment:
i think this needs to be `{{ template "sscdpa.psp.fullname" . }}`

Reply:
fixed

Reply:
This is still not fixed

Reply:
Done